ada_feeding/ada_feeding/trees/acquire_food_tree.py

```diff
@@ -73,9 +55,69 @@ def create_move_to_tree(
         -------
         tree: The behavior tree that acquires food
         """
+        # TODO: remove name and tree_root_name
+        # We have access to them implicitly via self.blackboard / self.blackboard_tree_root
```

Comment:
Good point, feel free to create an issue for this.

Currently, [`self.blackboard.namespace` and `self.blackboard_tree_root.namespace` are the ones that have the unchanged name](https://github.com/personalrobotics/ada_feeding/blob/74dab3aceba646fa5c770acda8c64ce03120bb80/ada_feeding/ada_feeding/trees/move_to_tree.py#L71). We should consider whether it should instead be `.name`.

Reply:
Made issue #111